### PR TITLE
Set custom request header on all axios requests for easier debugging

### PR DESCRIPTION
**Pull request by brandenrodgers**
## Description and Context

Depends on: https://github.com/HubSpot/hubspot-local-dev-lib/pull/123

Sets a custom request header that will apply to all requests. This makes it easier for us to debug issues in our backend systems because it lets us link incoming requests to certain CLI versions.

I'm open to suggestions for better naming on the header.

### CODE REVIEW

File: packages/cli/bin/cli.js [hunk 1]
```diff
@@ -110,9 +111,13 @@ const performChecks = argv => {
   }
 };
 
+const setRequestHeaders = () => {
+  addRequestHeader('X-Parent-User-Agent', `HubSpot CLI/${pkg.version}`);
```
Comment (m-roll) [inline packages/cli/bin/cli.js:115]: Is this a standard header?

Can this be `HubSpot CLI-lib/{version}`?

**Comment by brandenrodgers** [comment]
Relevant supporting PR: https://github.com/HubSpot/hubspot-local-dev-lib/pull/123

File: packages/cli/bin/cli.js (hunk 1 above)
Comment (brandenrodgers) [reply packages/cli/bin/cli.js]: Just to close the loop here, I updated this to use the existing `User-Agent` header (see my latest comment)

**Review by kemmerle — APPROVED** [review]
Tested and reviewed the code, everything is working great for me 👍

**Pull request merged**